tests/unit/store/unload-test.js

```diff
@@ -86,7 +86,7 @@ test("unload a record", function(assert) {
         store.unloadRecord(record);
       });
 
-      assert.equal(get(record, 'hasDirtyAttributes'), false, "record is not dirty");
```

Comment:
In `1.13`, it looks like if the assertion for `hasDirtyAttributes` is `undefined` when outside of the run-loop with `unloadRecord`. When inside the run-loop with `unloadRecord`, it returns `false`.